### PR TITLE
Subworkflow:traceback

### LINKED ISSUE #28: new module: genotype_variants

### LINKED ISSUE #39: New Subworkflow: Traceback
- [ ] add pvmaf modules 
- [ ] add genotype module
- [ ] add sub-workflow for traceback

### PR DESCRIPTION
**Pull request by buehlere**
## PR checklist

closes #39 

- [x] adding genotype variant module.
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [x] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [x] If necessary, include test data in your PR.
- [x] Remove all TODO statements.
- [x] Emit the `versions.yml` file.
- [x] Follow the naming conventions.
- [x] Follow the parameters requirements.
- [x] Follow the input/output options guidelines.
- [x] Add a resource `label`
- [x] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [x] nf-test test --profile=docker --tag <MODULE> 
  - [x] nf-test test --profile=singularity --tag <MODULE> 
  - [ ] nf-test test --profile=conda --tag <MODULE>

### CODE REVIEW

File: modules/msk/genotypevariants/all/main.nf [hunk 1]
```diff
@@ -0,0 +1,62 @@
+process GENOTYPEVARIANTS_ALL {
+    tag "$meta.id"
+    label 'process_single'
+
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'ghcr.io/msk-access/genotype_variants:sha-7e2c3c8a':
```
Comment (rnaidu) [inline modules/msk/genotypevariants/all/main.nf]: Should we add the released genotype_variants docker image before merging this PR into traceback?

**Review by rnaidu — CHANGES_REQUESTED** [review]
@buehlere thanks for working on this, everything looks great! I've added in the code to traceback to read in the input MAF file properly for genotypevariants_all. 

only change I'm requesting is for genotype_variants-- we should replace the docker image currently there with the official released version instead. Comment pointing this out is at [modules/msk/genotypevariants/all/main.nf](https://github.com/mskcc-omics-workflows/modules/pull/47/files/519e0055f802e5f647917d63069eafd76be589f1#diff-901a608517846dca240043fe57d6ea1878910d034a5ccaf0eb5f847f01c8ed6b)

**Review by rnaidu — APPROVED** [review]
Hi Eric, I tested out the subworkflow using nf-test and in the scope of nucleovar, and both tests are a success. nucleovar is also printing out the correct output MAF files we need. Thanks for your work on this!

File: tests/config/test_data.config [hunk 2]
```diff
@@ -723,6 +723,10 @@ params {
                 dbsnp_138_b37_chr17_vcf_gz                     = "${params.test_data_base_msk}/hg37/genomic_reference/GRCH37/vcf/dbsnp_138.b37.chr17.vcf.gz"
                 dbsnp_138_b37_chr17_vcf_gz_tbi                 = "${params.test_data_base_msk}/hg37/genomic_reference/GRCH37/vcf/dbsnp_138.b37.chr17.vcf.gz.tbi"
             }
+            'chr22' {
+                chr22maf                 = "${params.test_data_base_msk}/chr22/chr22/chr22.maf"
```
Comment (nikhil) [inline tests/config/test_data.config]: fairly minor but can you align the ='s like in the rest of the file

File: modules/msk/pvmaf/tagtraceback/main.nf [hunk 3]
```diff
@@ -0,0 +1,53 @@
+process PVMAF_TAGTRACEBACK {
+    tag "$meta.id"
+    label 'process_single'
+    // TODO nf-core: List required Conda package(s).
```
Comment (nikhil) [inline modules/msk/pvmaf/tagtraceback/main.nf]: Remove todo

**Review by nikhil — CHANGES_REQUESTED** [review]

File: modules/msk/pvmaf/tagtraceback/main.nf (hunk 3 above)
Comment (buehlere) [reply modules/msk/pvmaf/tagtraceback/main.nf]: removed!

File: tests/config/test_data.config (hunk 2 above)
Comment (buehlere) [reply tests/config/test_data.config]: fixed!

**Review by nikhil — APPROVED** [review]

**Pull request merged**